src/components/rangeslider/draw.js

```diff
@@ -352,7 +353,8 @@ function drawRangePlot(rangeSlider, gd, axisOpts, opts) {
 
         mockFigure.layout[oppAxisName] = {
             domain: [0, 1],
-            range: oppAxisOpts.range.slice()
+            range: oppAxisOpts.range.slice(),
+            calendar: oppAxisOpts.calendar
```

Comment:
That's fine :+1: 

By the way, I've been wanting to add support for multiple range sliders per plot for a while now. Would be a nice feature.